containers/src/Data/Map/Internal.hs

```diff
@@ -1800,7 +1800,7 @@ unionsWith f ts
 {-# INLINABLE unionsWith #-}
 #endif
 
--- | /O(m*log(n\/m + 1)), m <= n/.
+-- | \(O(m \log(n/m+1)), m \leq n\).
```

Comment:
I think we can take advantage of the better layout and so on MathJax offers us to fix the nagging partiality in this expression.

```suggestion
-- | \($O\bigl(m \log \bigl(\frac{n+1}{m+1}\bigr)\bigr), \quad m \leq n$\).
```

If you think a `\quad` space is too big, you can use a `\;` one. I'm not sure which is better.

Reply:
I changed it to `\(O(m \log\bigl(\frac{n+1}{m+1}\bigr)), \; m \leq n\)`, I hope that's okay? It keeps the O(...) consistent with the other annotations.

It looks like this:
![example](https://user-images.githubusercontent.com/44230978/163260565-0b7ee4d8-3daf-4544-bf75-10c99f2e463c.png)

Reply:
No, small parentheses around bigger ones look wrong. What's inconsistent?

Reply:
The inconsistency is that other O(...) annotations use normal parantheses instead of the big ones.

Reply:
That kind of inconsistency is just fine. Standard mathematical practice.

Reply:
Ok, I'll add the big parentheses.